hostscripts/clouddata/syncSLErepos

```diff
@@ -48,11 +48,17 @@ M=
 DRY=--dry-run
 DRY=
 
-rsync="$E rsync $DRY --timeout=1200 --fuzzy --bwlimit 10000
+rsync="$E rsync $DRY --timeout=1200 --fuzzy
     --delay-updates $rsync_delete -aH --no-owner --no-group $rsync_excludes "
 
 # http://download.suse.de/ibs/Devel:/Cloud:/4/images/*qcow2 -> images/SLES11-SP3-x86_64-cfntools.qcow2
 
+# prevent concurrent runs
+lockf=/srv/nfs/last-update.txt.lock
+
+lockfile -l $((4 * 60 * 60)) $lockf
```

Comment:
Shouldn't we exit the script here if this fails?

Reply:
man page says `If the number of retries is -1 (default, i.e., -r-1) lockfile will retry forever.` - so it would just wait here and not fail. but maybe we would want it to fail after some hours...

Reply:
Oh I see, `-l` is the timeout before it forces the lock, not the timeout before it gives up and fails.  Can we safely assume a sync will never take more than 4 hours?